docs/Bots.md

```diff
@@ -801,6 +801,40 @@ from your installation.
 
 * * *
 
+# SMTP Output Bot
+
+Sends a MIME Multipart message containing the text and
```

Comment:
Add a text that this will send an email for *every individual* event that it receives